source/stylesheets/components/_article.scss

```diff
@@ -4,4 +4,43 @@ article {
   @include media($large-screen) {
     @include span-columns(8.5);
   }
+
+  footer {
+    @include clearfix;
+    border-top: $base-border;
+    margin-top: 5em;
+    padding: $base-spacing 0;
+
+    .previous-guide {
+      float: left;
+
+      &::before {
+        content: "‹";
```

Comment:
How do you feel about using the escaped unicode for these? I understand it's more recognizable when using the font but some editors and fonts will screw it up.